.github/workflows/vscodeSupportE2E.yml

```diff
@@ -0,0 +1,111 @@
+name: VS Code Support E2E Tests
+on:
+  workflow_run:
+    workflows:
+      - Nightly Build Develop
+    types:
+      - completed
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+      testToRun:
+        description: 'Run this E2E test'
+        required: false
+        default: 'anInitialSuite.e2e.ts'
+        type: string
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: false
+        type: string
+      vscodeVersion:
+        description: 'VS Code Version'
+        required: false
+        type: string
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        nodeVersion:
+          - 18.11.0
```

Comment:
Why Node v18.11.0 instead of v18.18.2?

Reply:
I had just used the same version we're using in the `runE2ETest.yml` workflow, and hadn't realized we had updated the version in the `.nvmrc` file. I just updated both workflows with v18.18.2. Thanks!